lib/gooddata/bricks/rollout_brick.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'brick'
+
+module GoodData::Bricks
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.